daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-template.md

```diff
@@ -0,0 +1,93 @@
+---
+type: docs
+title: MapR template file
+linkTitle: MapR template
+weight: 2000
+description: Unpack the MapR template file and its properties
+---
+
+{{% alert title="Note" color="primary" %}}
+ MapR (Multi-app Run using `dapr run -f`) is currently a preview feature only supported in Linux/MacOS. 
+{{% /alert %}}
+
+The MapR template file is a YAML file that you can use to run multiple applications at once. Execute the following CLI command to read the MapR template file, named `dapr.yaml` by default:
+
+```cmd
+// the template file needs to be called `dapr.yaml` by default
+
+dapr run -f
+```
+
+If the MapR template file is named something other than `dapr.yaml`, run:
+
+```cmd
+dapr run -f ./<your-preferred-file-name>.yaml
+```
+
+The MapR template file can include the following properties. Below is an example template showing two applications that are configured with some of the properties. 
+
+```yaml
+version: 1
+common: # optional section for variables shared across apps
+  resourcesPath: ./app/components # any dapr resources to be shared across apps
+  env:  # any environment variable shared across apps
+    - DEBUG: true
+apps:
+  - appID: webapp # required
```

Comment:
correction: appID is not a required parameter. If it is not provided, it will be derived from `appDirPath`. In short, in the template file only the `appDirPath` is the only mandatory param.